xgcm/test/datasets.py

```diff
@@ -144,6 +348,44 @@
 }
 
 expected_values = {
+    "nonperiodic_sgrid2D": {
+        "axes": {
+            "X": {"center": "XC", "left": "XG"},
+            "Y": {"center": "YC", "left": "YG"},
+        }
+    },
+    "periodic_sgrid2D": {
+        "axes": {
+            "X": {"center": "XC", "left": "XG"},
+            "Y": {"center": "YC", "left": "YG"},
+        }
+    },
+    "xperiodic_sgrid2D": {
+        "axes": {
+            "X": {"center": "XC", "left": "XG"},
+            "Y": {"center": "YC", "left": "YG"},
+        }
+    },
+    "yperiodic_sgrid2D": {
+        "axes": {
+            "X": {"center": "XC", "left": "XG"},
+            "Y": {"center": "YC", "left": "YG"},
+        }
+    },
+    "sgrid2Dvert": {
+        "axes": {
+            "X": {"center": "XC", "outer": "XG"},
+            "Y": {"center": "YC", "outer": "YG"},
+            "Z": {"center": "vcell", "inner": "vnode"},
+        }
+    },
+    "sgrid3D": {
+        "axes": {
+            "X": {"center": "XC", "right": "XG"},
+            "Y": {"center": "YC", "right": "YG"},
+            "Z": {"center": "kcell", "left": "knode"},
+        }
+    },
```

Comment:
This actually already contains all the expected values I mentioned in my example above. Refactoring these into separate tests should be fairly straightforward.